spec/features/two_factor_authentication/multiple_mfa_sign_up_spec.rb

```diff
@@ -136,6 +137,31 @@
     end
   end
 
+  scenario 'when backup codes are the only selected option' do
+    sign_in_before_2fa
+
+    expect(current_path).to eq authentication_methods_setup_path
+
+    click_2fa_option('backup_code')
+
+    click_continue
+
+    expect(current_path).to eq backup_code_setup_path
+
+    click_continue
+
+    expect(page).to have_link(t('components.download_button.label'))
+
+    click_continue
+
+    expect(page).to have_current_path(
+      auth_method_confirmation_path,
+    )
+
+    click_button t('mfa.skip')
+    expect(page).to have_current_path(confirm_backup_codes_path)
```

Comment:
Can we expand this spec to describe the behaviors expected for each of the button options?